### DISCUSSION TITLE
user guide

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#3270] [Ideas]
Maybe it's worth putting the user guide into the Transifex platform, so it will be accessible in languages other than English?

**Comment by alex-w** [comment]
It’s not possible by technical reasons

**Comment by gzotti** [comment]
This is far from easy. When Stellarium was still new (around 2007/08?) there were a few non-English translations. However, usually people don't continue/update such works. 

It would take high effort to make it auto-translatable (accessible for Transifex) in the first place. Translators will need to know some LaTeX. 

We have once tried to produce an HTML online version of the guide from the same sources. However, my knowledge of these translation tools, and of CSS, is too limited to make that attractive. Such websites could be read by online translation services (available as web browser plugin). If somebody wants to play with this, please show us a better HTML version!
